src/Microsoft.Graph.Cli.Core/IO/ConsoleUtilities.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Security;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Graph.Cli.Core.IO;
+
+public static class ConsoleUtilities
+{
+    public static string ReadPassword(string? message = null)
+    {
+        var pass = new StringBuilder();
```

Comment:
if the logic is the same as the async implementation, why not just call the async method + GetAwaiter and GetResult ?

Reply:
I didn't think that was the preferred option. The .NET framework offers both methods in its libraries